ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.13-nightly-x86.yaml

```diff
@@ -23,7 +23,7 @@ resources:
       memory: 200Mi
 tests:
 - as: control-plane-etcd-fips-24nodes
-  cron: 0 8 16 * *
+  cron: 0 8 23 * *
```

Comment:
This is the same expression as the gcp-4.12 job.

I suggest: `0 17 23 * *`

Reply:
Done